tests/WP_Ultimo/Gateways/Stripe_OAuth_Test.php

```
@@ -0,0 +1,298 @@
+
+		// OAuth should take precedence
+		$this->assertEquals('oauth', $gateway->get_authentication_mode());
+		$this->assertTrue($gateway->is_using_oauth());
+	}
+
+	/**
+	 * Test OAuth authorization URL generation via proxy.
+	 */
+	public function test_oauth_authorization_url_generation() {
+		// Mock proxy response
+		add_filter('pre_http_request', function($preempt, $args, $url) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$args'. (undefined)

(UnusedFormalParameter)